pybamm/solvers/jax_solver.py

```diff
@@ -30,9 +30,9 @@ class JaxSolver(pybamm.BaseSolver):
 
     Parameters
     ----------
-    method: str
+    method: str, optional (see :func:`jax.experimental.odeint` for details)
         'RK45' (default) uses jax.experimental.odeint
-        'BDF' uses custom jax_bdf_integrate (see jax_bdf_integrate.py for details)
+        'BDF' uses custom jax_bdf_integrate (see :func:`jax_bdf_integrate.py` for details)
```

Comment:
This should be something like:
```
    method: str, optional (see :func:`jax.experimental.odeint` for details)
        * 'RK45' (default) uses jax.experimental.odeint
        * 'BDF' uses custom jax_bdf_integrate (see :func:`jax_bdf_integrate.py` for details)
```

Reply:
Thanks a lot @kratman

Reply:
Unresolving the conversation, this doesn't work either: https://pybamm--3994.org.readthedocs.build/en/3994/source/api/solvers/jax_solver.html

because `jax_bdf_integrate.py` is a Python file so we can't reference it directly. We can link to `jax_bdf_integrate`, though. The `jax.experimental.odeint` reference isn't being highlighted either, we can use 

````rst
:ref:`jax.experimental.odeint`
````

and that should work.

Reply:
I did not realize the link was broken, I thought you were just changing where it pointed. My mistake

Reply:
I think you unresolved the wrong one

Reply:
It doesn't work, though?

Reply:
Never mind, found the issue